sdk/transform/chunker.go

```diff
@@ -0,0 +1,73 @@
+package transform
+
+import (
+	"github.com/aserto-dev/ds-load/sdk/common/js"
+	"github.com/aserto-dev/ds-load/sdk/common/msg"
+	v2 "github.com/aserto-dev/go-directory/aserto/directory/common/v2"
+)
+
+type Chunker interface {
```

Comment:
I don't think this interface needs to be defined here. In Go, interfaces are typically defined by the _user_ not the _implementer_.
If the `transform` struct in `sdk/transform/transform.go` wants to be able to use any type that has a `WriteChunks` method with the appropriate signature then _it_ should define the interface it expects.

But unless we plan to have multiple Chunker implementations, I'm not sure we even need an interface at all.

Reply:
But given the conversation we had about chunking today, maybe we don't need this chunker at all.